### ISSUE TITLE
500 error when changing category of node_type

### ISSUE THREAD

**Issue by holgerd** [sozialhelden/wheelmap#404]
- I'm logged in as admin on https://wheelmap.org/admin/node_types/41/edit . I change the category of museum from "education" to "tourism" and click save
- I expect to see a success message
- Instead, I see "something went wrong" and the url is https://wheelmap.org/admin/node_types/41

**Comment by 1000miles** [comment]
Thanks for reporting this, @holgerd! :)

After a short research it throws same error as mentioned in #365 
`ActiveModel::ForbiddenAttributesError (ActiveModel::ForbiddenAttributesError)`

We will make sure to check all pages on the admin dashboard to fix this issue. Thanks for your patience.

**Comment by 1000miles** [comment]
@holgerd 

cc @schultyy 

So we have checked all pages on the active_admin dashboard of the production server and here are the results:

- `/admin/providers` => ERROR: `ActiveModel::ForbiddenAttributesError (ActiveModel::ForbiddenAttributesError)`  => Fixed in PR https://github.com/sozialhelden/wheelmap/pull/394
- `/admin/provided_pois` => ERROR: `ActiveModel::ForbiddenAttributesError (ActiveModel::ForbiddenAttributesError)`  => Fixed in PR https://github.com/sozialhelden/wheelmap/pull/394

----
- `/admin/addresses` => create,  edit & delete entry works!
- `/admin/admin_users` => create,  edit & delete entry works!
- `/admin/users`  => create,  edit & delete entry works!
- `/admin/categories` => ERROR: `ActiveModel::ForbiddenAttributesError (ActiveModel::ForbiddenAttributesError)` => Fix on the way
- `/admin/node_types` => ERROR: `ActiveModel::ForbiddenAttributesError (ActiveModel::ForbiddenAttributesError)`  => Fix on the way
- `/admin/photos` => ERROR:  `WARNING: Can't mass-assign protected` => Reported in Ticket #415
- `/admin/regions` => ERROR:  `ActionView::Template::Error` => Reported in Ticket #414

**Comment by 1000miles** [comment]
@holgerd The fix is deployed on staging and ready for review.

**Comment by holgerd** [comment]
@1000miles  I tested this. It does not give me a 500 error but two things came up:

1. 
tourism exists twice on staging in the dropdown. I tested with both.

2. 
- I changed the category from "Bildung" to "Tourismus" and clicked "Node type aktualisieren" and then went to https://staging.wheelmap.org/nodes/new/name_category to see where "museum" shows up.
- I expected it to be in the new category "Tourismus"
- Instead, it is still in "Bildung"
